### DISCUSSION TITLE
Asynchronous File Loading

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#188] [Answers & Questions] [answered]
Hi,
is there in cables the possibility of "Asynchronous File Loading" on board?
That is, that not all files have to be downloaded before the patch can start, but that the files are downloaded one after the other?

My application would be multichanel .opus files. These are large and would take a long time to download at startup. But of course this applies to all large files - whatever their format.

Thank you!

**Comment by pandrr** [comment, marked as answer]
all files are loaded asynchronously in cables, by default the patch starts already without any files loaded.
only loadingstatus op waits for all files finished loading.

if you want to load a new file at any point in time, you can dynamicly change the filename in the audio op. e.g. SwitchFile to start loading a bigger file.

**Comment by inst5** [comment]
Ah great - thanks pandrr for that quick feedback!!
